core/src/main/java/org/opensearch/sql/expression/window/WindowFunctions.java

```diff
@@ -34,6 +38,8 @@ public void register(BuiltinFunctionRepository repository) {
     repository.register(rowNumber());
     repository.register(rank());
     repository.register(denseRank());
+    repository.register(brain());
+    repository.register(simplePattern());
```

Comment:
I'm thinking are these new algorithm real window function? If users specify `order by` or `partition by`, does it still generate meaningful result?